Application/EdFi.Ods.Api/Helpers/AssemblyLoaderHelper.cs

```diff
@@ -143,16 +143,19 @@ public static IEnumerable<string> FindPluginAssemblies(string pluginFolder, bool
                     continue;
                 }
 
-                string extensionFolder = Path.GetDirectoryName(assemblyPath);
-                var validationResult = validator.ValidateObject(extensionFolder);
-
-                if (!validationResult.Any())
-                {
-                    yield return assembly.Location;
-                }
-                else
+                if (assemblyPath.Contains("Extensions"))

```

Comment:
This validation needs to be reworked rather than relying on naming conventions.  Types of plugins can be identified based on the assemblyMetadata.json file, but right now it looks like that file isn't being embedded in the plugins currently being built.  Given how close to release we are, we'll probably have to run with this naming convention check, and spin up another ticket to rework this validation to rely on identifying the assembly type using the same approach as code gen.

Reply:
Alternatively we might be able to extend the usage of IPluginMarker to introduce more specific IExtensionsPluginMarker and IProfilesPluginMarker, but I'm hesitant on having multiple ways to identify an assembly type, preferably everything would rely on the same approach.

Reply:
@blmeyers , @vimayya   I created new ticket https://tracker.ed-fi.org/browse/ODS-4664